guides-and-standards/standard-public-repos.md

```diff
@@ -0,0 +1,45 @@
+# Standards for public GitHub repositories at Arcadia
+
+This document describes the standards for public GitHub repositories at Arcadia. These standards are intended to ensure that the code we release is well-documented, reproducible, and easy to use. All Arcadia GitHub repositories should meet these standards prior to being made public.
+
+## Aside about repository templates
+
+We have Arcadia-specific templates for [Python](https://github.com/Arcadia-Science/python-analysis-template), [Snakemake](https://github.com/Arcadia-Science/snakemake-template), and [R](https://github.com/Arcadia-Science/r-analysis-template) repositories. It is strongly recommended to start new projects using these templates, as this will mean that your repository will automatically satisfy many of the requirements below.
+
+## Standards for organization and documentation
+
+- **Repository organization**: The repository should be well-organized with descriptive file names and a clear directory structure.
+  - Scripts, notebooks, and data files should generally be in separate directories.
+  - Directory names and filenames should not include spaces or special characters.
+  - Avoid very long scripts or notebooks. If a script or notebook is more than 500-1000 lines long, consider breaking it up into smaller scripts or functions.
+- **Descriptive README**: The repository should include a thorough and clear README that explains the context and purpose of the repo, what the code does, and how to use it.
+  - The README should explain how to create the same environment that was originally used to run the code. This includes the operating system you ran the code on, the software you installed, and the version of the software.
+  - The README should include an explanation of how to access or prepare any data needed to run the code.
+- **Dependency versions**: The exact versions of all direct dependencies (i.e., Python and/or R packages) used in the repository should be listed. Typically, this takes the form of a Conda environment file (usually `envs/dev.yml`).
+  - For Python projects, dependencies can also be listed in a `requirements.txt` file or in `pyproject.toml` if you are using a package manager like `poetry`.
```

Comment:
I'd maybe note `poetry` is preferred. Or note that these dependencies can also be managed by conda